examples/pubsub/server_pubsub_subscribe_monitoring_linux.c

```diff
@@ -0,0 +1,539 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+/**
+ * IMPORTANT ANNOUNCEMENT
+ * The PubSub Subscriber API is currently not finished. This example can be used
+ * to receive and display values that are published by tutorial_pubsub_publish
+ * example in the TargetVariables of Subscriber Information Model.
```

Comment:
Which parts of the internal API are used here? I think there is no reason for 'The PubSub Subscriber API is currently not finished.'. This hint is only for the subscribe-standalone example where the complete message flow is skipped by calling the receive function on the user-level.

Reply:
ok, removed the sentence